src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-page.component.ts

```diff
@@ -382,8 +384,20 @@ export class InstructorSessionResultPageComponent extends InstructorCommentsComp
           )
       ;
 
-      response.subscribe(() => {
-        this.router.navigateByUrl('/web/instructor/sessions');
+      response.subscribe((res: FeedbackSession) => {
+        this.session = res;
+        if (this.session.resultVisibleFromTimestamp) {
+          this.formattedResultVisibleFromTime = this.timezoneService
+            .formatToString(this.session.resultVisibleFromTimestamp, this.session.timeZone, this.TIME_FORMAT);
+        } else {
+          this.formattedResultVisibleFromTime = 'Not applicable';
+        }
+        if (isPublished) {
```

Comment:
Are the conditionals `isPublished` and `this.session.resultVisibleFromTimestamp` equivalent?